src/applications/vaos/sass/vaos.scss

```diff
@@ -45,4 +45,11 @@
   .form-radio-buttons label {
     max-width: none;
   }
+}
+
+.vaos-form__facility-type {
+  legend {
+    margin-bottom: 30px;
```

Comment:
Would prefer to use a utility class for this, but not sure if there's a way to add a class to a `ui:title`/`legend` in uiSchema.

Reply:
I think you can pass a component to the `ui:title` prop, that might get you what you need.